hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/HttpServer2.java

```diff
@@ -49,6 +49,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
nit:
the import should be in the below import block with other `com.google` import statement in alphabetical order

Reply:
Thanks for the comment! Addressed.